### DISCUSSION TITLE
Convert to monorepo so we can export rn-primitives on npm

### DISCUSSION THREAD

**Discussion by panzacoder** [founded-labs/react-native-reusables#44] [Ideas]
Title is the gist, I know moving to another repo is in your roadmap, perhaps it would be less work to convert this to a monorepo so you can continue to develop in parallel, but can also set up releases for the rn-primitives package.

Thoughts? If you're in favor I'd be glad to take a stab at the re-org.

**Comment by mrzachnugent** [comment]
Hi @panzacoder, it was definitively considered. 

The main reason is that it will be moved to a Open-source Github account from the team that maintains [create-expo-stack](https://createexpostack.com/). I do not have any open-source npm package experience, they are great developers, and I'm excited to team up with them.

**Reply by panzacoder** [reply]
Good to hear! 

I have pulled all of these components into a "package" in my own monorepo for now, if anyone is looking for contributors for supporting this transition I would love to help!

**Reply by mrzachnugent** [reply]
@panzacoder I'm reconsidering this as an option. Is your repo public? I would love to have a look at it.

**Reply by panzacoder** [reply]
I have it in an incomplete state (needed to migrate some stuff to nativewind v4, address some TS issues), I'll try to pull it into my public starter soon, that should help me isolate issues also.

**Reply by mrzachnugent** [reply]
Don't go through too much trouble, I was just asking if you had one readily available. I wouldn't want you to put in work for something that I won't use.

**Reply by panzacoder** [reply]
👍 I already did the nativewind migration, just debugging some tsconfig stuff. I think moving it to the starter will be helpful for me also. I'll post here when I have time to get through it.
